thermx.tex

```diff
@@ -295,7 +295,7 @@ \subsection{Incoherent Inelastic Scattering}
 with the point at zero and the first point above zero that can be derived
 from the positive and negative values of $\beta$ from the evaluator's
 $\beta$ grid using Eq.~\ref{ebeta}.  (For free-gas scattering, the
-$\beta$ grid is taken to have nine entries between 0.0 eV and 25.0 eV).
+$\beta$ grid is taken to have 45 entries between 0.0 and 3500).
```

Comment:
You lost the units on those numbers.

Reply:
beta is actually dimensionless (it is defined as the energy transfer divided by kT) so I corrected it.